nvflare/private/fed/server/sys_cmd.py

```diff
@@ -131,7 +130,7 @@ def _process_replies(self, conn, replies):
             else:
                 conn.append_string(": No replies")
 
-    def report_resources(self, conn: Connection, args: List[str]):
+    def report_resources(self, conn: Connection, args: [str]):
```

Comment:
we should keep the type hints in this file

Reply:
The List[str] and [str] essentially the same. It's a string list.

Reply:
From type hinting tool static analysis perspective (for example "mypy"),
Only List[str] is valid for the check.

otherwise we get 
`error: Bracketed expression "[...]" is not valid as a type  [valid-type]
`

Reply:
Added back